src/mentionPlugin/MentionSearch/index.js

```diff
@@ -48,6 +54,7 @@ export default (mentions) => {
               ))
             }
           </Dropdown>
+          : void 0}
```

Comment:
That is actually more of habit, I will change that to undefined. Also we actually do not need a separate dropdown component, here is not doing anything more then applying styles to a single div.